highfidelity/hifi
# 6443 Audio Menu Sample Sound won't play while domain is loading

This PR preloads the sample audio when the application starts so that it's always available by the time the menu loads.

https://highfidelity.fogbugz.com/f/cases/6443

...

TEST PLAN

- Go to Welcome or any other domain with a lot of content
- While it is loading, open the audio menu and play the sample audio
- Activate the advanced and developer menus and go to edit ->clear cache
- Repeating this process should have the same outcome. (sample audio always plays)
